src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -546,9 +551,7 @@ public String getRequestVariableName(Method method) {
   /////////////////////////////////////// Type names /////////////////////////////////////////////
 
   protected String getInterfaceName(InterfaceConfig interfaceConfig) {
-    return interfaceConfig.hasInterfaceNameOverride()
-        ? interfaceConfig.getInterfaceNameOverride()
-        : interfaceConfig.getInterface().getSimpleName();
+    return interfaceConfig.getName();
```

Comment:
Does this method still need to exist now that it's just an alias for another method?